.travis.yml

```diff
@@ -26,25 +27,30 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.2.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - SYMFONY_PHPUNIT_VERSION=5.7
-    - TEST_INSTALLATION=false
+    - SYMFONY_PHPUNIT_VERSION=7
+    - PHPUNIT_VERSION=7
```

Comment:
i do not really know, i tried to replace simple-phpunit by the plain version, where i got real trouble, so i would think the first is enough, but this would be a change i can push through our dev-kit

Reply:
i would keep simple-phpunit. the main reason for that thing was when phpunit had different incompatible version of a symfony component and that led to a mess.